### PR TITLE
[MeshingApplication] Cleanup meshing utilities. Removing method moved to the core weeks ago

### PR DESCRIPTION
**Pull request by loumalouomega**
Basically that

### CODE REVIEW

File: kratos/python/add_utilities_to_python.cpp [hunk 1]
```diff
@@ -422,6 +422,32 @@ std::string GetRegisteredNameCondition(const Condition& rCondition)
...
+    rAuxiliarModelPartUtilities.EnsureModelPartOwnsProperties();
+}
+
+void EnsureModelPartOwnsPropertiesWithClear(
+    AuxiliarModelPartUtilities& rAuxiliarModelPartUtilities,
+    const bool RemovePreviousProperties
+    )
+{
+    rAuxiliarModelPartUtilities.EnsureModelPartOwnsProperties(RemovePreviousProperties);
+}
```
Comment (rubenzorrilla) [inline kratos/python/add_utilities_to_python.cpp]: As a comment, I think that at some point we should start promoting the use of lambdas for this.

**Review by rubenzorrilla — DISMISSED** [review]

File: kratos/python/add_utilities_to_python.cpp (hunk 1 above)
Comment (loumalouomega) [reply kratos/python/add_utilities_to_python.cpp]: Indeed

**Comment by loumalouomega** [comment]
@roigcarlo can we deactivate the requirement of CI windows until the problem is solved?, we still have appveyor to check that

**Comment by philbucher** [comment]
Appveyor doesn’t run the tests
We will find a solution today, sorry for he delays and inconvenience

File: kratos/python/add_utilities_to_python.cpp (hunk 1 above)
Comment (philbucher) [reply kratos/python/add_utilities_to_python.cpp]: I think I have a better proposal for this: [pybinds `overload_cast`](https://pybind11.readthedocs.io/en/stable/classes.html)

If I understood it correctly then we don't need any lambdas or wrapper functions

Downside: its C++14 :(

I think it is still worth waiting, bcs then we don't have to do the work twice and all of our python-interfaces will become much cleaner :D

Comment (rubenzorrilla) [reply kratos/python/add_utilities_to_python.cpp]: I wasn't aware of this feature.

In any case, I wouldn't forbid the use of lambdas for this. Of course, not to change whatever it is already there, but at least for the new exported methods.

Comment (philbucher) [reply kratos/python/add_utilities_to_python.cpp]: > Of course, not to change whatever it is already there

Thats basically what I meant ;)

Comment (loumalouomega) [reply kratos/python/add_utilities_to_python.cpp]: I added the lambdas, it compiles with C++11, if you don't like it, it can be reverted

**Review by philbucher — APPROVED** [review]

**Pull request merged**